library/objective-c/EnvoyEngine.h

```diff
@@ -88,6 +99,20 @@ extern const int kEnvoyFilterTrailersStatusResumeIteration;
 
 @property (nonatomic, copy) NSArray * (^onResponseTrailers)(EnvoyHeaders *trailers);
 
+@property (nonatomic, copy) void (^setRequestFilterCallbacks)
+    (id<EnvoyHTTPFilterCallbacks> callbacks);
+
+@property (nonatomic, copy) NSArray * (^onResumeRequest)
```

Comment:
I know adding type annotations here isn't possible since they're varied types, but can we at least document the expected contents and lengths of these `NSArray`s?

Reply:
Sounds good.